src/core/eme/clean_old_loaded_sessions.ts

```diff
@@ -0,0 +1,78 @@
+/**
```

Comment:
Note: I also moved the `cleanOldLoadedSessions` function to its own file to stay consistent.